card/lib/card/env/location.rb

```diff
@@ -12,7 +12,7 @@ def card_path rel_path
         if rel_path =~ %r{^(https?\:)?/}
           rel_path
         else
-          "#{Card.config.relative_url_root}/#{rel_path}"
```

Comment:
if you've already loaded Card, then Card.config is still an acceptable alternative to Cardio.config, right?  And if you're in a subclass of Card, you've already loaded Card, right?

Reply:
I think Card.config will probably be the more approachable convention when already inside card context.